atsign-foundation/at_app_cli
# Support for desktop and Namespace Validation

closes #31 
closes #16 

**- What I did**
- Used at_utils to validate the namespace arugment
- Updated to use the at_app_flutter version with support for flutter_dotenv
- Removed flutter_config and its setup steps

**- How I did it**

- Local changes

**- How to verify it**

Pull the prerelease with ```flutter pub global activate at_app 4.0.0-rc+2```

Then try it out, it should support desktop and the namespace argument should only accept valid @‎signs.

This also verifies [the at_app_flutter PR](https://github.com/atsign-foundation/at_widgets/pull/258)

**- Description for the changelog**
Support for desktop and Namespace Validation
